app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+
+    if len(command_parts) != 3 or command_parts[0] != "mv":
+        return
+
+    old_file_path = command_parts[1]
+    new_file_path = command_parts[2]
+
+    new_path_parts = os.path.split(new_file_path)
+
+    if not new_path_parts[0]:
+        os.rename(old_file_path, new_file_path)
+        return
+
+    new_file_name = new_path_parts[1]
+
+    os.makedirs(new_path_parts[0], exist_ok=True)
+
+    with open(old_file_path, "r") as file:
+        content = file.read()
+
+    path_to_move = new_file_path if new_file_name else (new_file_path
```

Comment:
Fixed. But this code wouldn't work for this case: "If name in destination path ends with / it must be considered as a directory", because here we don't have the file name in the destination path. There is no test for this case so tests are passed successfully. But there is such case in the task.